src/presenters/breadcrumbs-presenter.php

```diff
@@ -137,8 +137,13 @@ protected function crumb_to_link( $breadcrumb, $index, $total ) {
 			// If it's not the last element and we have a url.
 			$link      .= '<' . $this->get_element() . '>';
 			$title_attr = isset( $breadcrumb['title'] ) ? ' title="' . \esc_attr( $breadcrumb['title'] ) . '"' : '';
-			$link      .= '<a href="' . \esc_url( $breadcrumb['url'] ) . '"' . $title_attr . '>' . $text . '</a>';
-			$link      .= '</' . $this->get_element() . '>';
+			$link      .= '<a';
+			// If the block is rendered server side (via a rest request) make sure the links target blank.
```

Comment:
Why _only_ for rest requests? I think the reason why would be the interesting part of a comment.